x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/RoleDescriptor.java

```diff
@@ -396,6 +415,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params, boolea
         if (hasRestriction()) {
             builder.field(Fields.RESTRICTION.getPreferredName(), restriction);
         }
+        if (description != null) {
+            builder.field(Fields.DESCRIPTION.getPreferredName(), description);
+        }
```

Comment:
I don't think it should, for the same reasons we don't include `name` and `metadata`. This is because a set of privileges a user has is the union of multiple roles. There is potentially no single description (or name) to return.